miner/worker.go

```diff
@@ -1336,10 +1337,23 @@ LOOP:
 	// when in-turn, compare with remote work.
 	from := bestWork.coinbase
 	if w.bidFetcher != nil && bestWork.header.Difficulty.Cmp(diffInTurn) == 0 {
+		waitBidTimer := time.NewTimer(w.config.DelayLeftOver / 5)
```

Comment:
can you check the real `DelayLeftOver` config on validator? Is it zero?

Reply:
fixed